wagtail/admin/templates/wagtailadmin/login.html

```diff
@@ -59,12 +59,18 @@ <h1>{% block branding_login %}{% trans "Sign in to Wagtail" %}{% endblock %}</h1
                                     </li>
                                 {% endfor %}
                             {% endblock extra_fields %}
-
                             <li class="checkbox">
-                                <div class="field">
-                                    <label><input name="remember" type="checkbox" />{% trans "Remember me" %}</label>
-                                </div>
+                                <span>
+                                    <label><input name="remember" type="checkbox" />{% trans "Remember me" %} {% if show_password_reset %}
+                                        <span class="help">
+                                            <a class="help__link" href="{% url 'wagtailadmin_password_reset' %}" aria-label="{% trans 'Reset your password' %}">{% trans "Forgot your password?" %}</a>
+                                        </span>
+                                    {% endif %}</label>
```

Comment:
Try to ensure your Django template nesting is neat. It's hard to read this change - where did the if start?